content_api/publish/service.py

```diff
@@ -45,6 +45,10 @@ def publish(self, item, subscribers=[]):
             doc.setdefault('firstcreated', now)
             doc.setdefault('versioncreated', now)
             doc.setdefault(config.VERSION, item.get(config.VERSION, 1))
+            item[ASSOCIATIONS] = doc.get(ASSOCIATIONS, {}).copy()
+            for _, assoc in item[ASSOCIATIONS].items():
+                assoc['subscribers'] = [str(subscriber[config.ID_FIELD]) for subscriber in subscribers]
```

Comment:
and here would put probably `assoc.setdefault('subscribers', subscribers)`

Reply:
right probably without the `str` call which should be handled by assign_associations without that condition